stream-chat-android-state/src/main/java/io/getstream/chat/android/offline/plugin/state/global/GlobalState.kt

```diff
@@ -34,16 +34,19 @@ public interface GlobalState {
     /**
      * The current user in the OfflinePlugin state.
      */
+    @Deprecated("Use ClientState.user")
```

Comment:
What do you think about adding the `replaceWith` parameter for letting people migrate to new interfaces easily?